packages/presentation/pubspec.yaml

```diff
@@ -15,10 +15,13 @@ dependencies:
 
   flutter_bloc: ^8.1.1
```

Comment:
u don't beed flutter_bloc anymore
and u don't need import of get_it. U should to get it instance through method in app_injector.dart

Reply:
removed